oort/oort-web/src/main/groovy/com/netflix/spinnaker/oort/controllers/ProjectController.groovy

```
@@ -0,0 +1,256 @@
+import com.netflix.spinnaker.oort.model.ServerGroup
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.MessageSource
+import org.springframework.context.i18n.LocaleContextHolder
+import org.springframework.http.HttpStatus
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor
+import org.springframework.stereotype.Component
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import rx.Observable
+import rx.Scheduler
+import rx.schedulers.Schedulers
+
+import com.netflix.spinnaker.oort.model.ServerGroup.InstanceCounts as InstanceCounts
+
+@Slf4j
+@RestController
+@RequestMapping("/projects/{project}")
+class ProjectController {
+
+  private final Scheduler queryClusterScheduler
+
+  @Autowired
+  ProjectController(@Value('${threadPool.queryCluster:25}') int threadPoolSize) {
+    this(Schedulers.from(newFixedThreadPool(threadPoolSize)))
+  }
+
+  ProjectController(Scheduler queryClusterScheduler) {
+    this.queryClusterScheduler = queryClusterScheduler
+  }
+
+  @Autowired
+  Front50Service front50Service
+
+  @Autowired
+  MessageSource messageSource
+
+  @Autowired
+  List<ClusterProvider> clusterProviders
+
+  @RequestMapping(method= RequestMethod.GET, value = "/clusters")
+  List<ClusterModel> getClusters(@PathVariable String project) {
+    Map projectConfig = null
+    try {
+      projectConfig = front50Service.getProject(project)
+    } catch (e) {
+      log.error("Unable to fetch project (${project})", e)
+      throw new ProjectNotFoundException(name: project)
```

Slightly different over there in that it's coming from a local provider vs. this one is going over the wire to front50. 

In theory we should just propagate the Retrofit error upwards, but we don't really have anything generic extracted to do that.

Guess this is fine for now.